wp-content/themes/haruoshi_origin/src/scss/style.scss

```
@@ -1,3 +1,30 @@
+:root {
+    --header-height: 10%;
+    --footer-height: 3%;
+  }
+
 body {
-    background-color: coral;
+    height: 2000px;
+    position: relative;
+    background-color: pink;
```

```suggestion
    position: relative;
```
できるだけ要素セレクターは使いたくないから、heightとbackground-colorの指定は.containerの中でしよう。